pandas/tests/io/parser/common.py

```diff
@@ -238,6 +238,17 @@ def test_csv_mixed_type(self):
         out = self.read_csv(StringIO(data))
         tm.assert_frame_equal(out, expected)
 
+    def test_csv_index_col_and_nrows(self):
+        data = """A,B,C
```

Comment:
Can you add a comment with the GitHub issue number here?

Reply:
Also, I think it needs a more descriptive test name - how about `test_read_csv_low_memory_no_index_cols_rows`?